test/Oscoin/Test/Consensus/Nakamoto.hs

```diff
@@ -88,7 +86,7 @@ instance (IsCrypto c) => TestableNode c PoW (NakamotoNode c) (NakamotoNodeState
         let stateStore = mkStateHashStore nakStateStoreL
         let receiptStore = mkStateReceiptStore nakReceiptStoreL
         let ledger = Ledger.mkLedger (fst blockStore) stateStore dummyEvalBlock receiptStore
-        maybeBlock <- mineBlock ledger nakConsensus tn defaultBeneficiary
+        maybeBlock <- mineBlock ledger nakConsensus tn someBeneficiary
```

Comment:
I find the notion of `some` a bit .. vague perhaps? What about `testBeneficiary` or `mockBeneficiary`? At least now the purpose and intention in clear. I suspect the same could apply for `someSealedGenesisBlock` & co.